src/backend/oracle-data-api/src/test/java/ca/bc/gov/open/jag/tco/oracledataapi/service/DisputeServiceOrdsOccamTest.java

```diff
@@ -14,19 +16,31 @@
 import org.apache.commons.lang3.time.DateFormatUtils;
 import org.assertj.core.util.Arrays;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 
 import ca.bc.gov.open.jag.tco.oracledataapi.BaseTestSuite;
 import ca.bc.gov.open.jag.tco.oracledataapi.model.Dispute;
+import ca.bc.gov.open.jag.tco.oracledataapi.ords.occam.api.HealthApi;
+import ca.bc.gov.open.jag.tco.oracledataapi.ords.occam.api.model.PingResult;
 import ca.bc.gov.open.jag.tco.oracledataapi.util.RandomUtil;
 
-@ConditionalOnProperty(name = "repository.dispute", havingValue = "ords", matchIfMissing = true)
-class DisputeServiceOrdsTest extends BaseTestSuite {
+@EnabledIfEnvironmentVariable(named = "DISPUTE_REPOSITORY_SRC", matches = "ords")

```

Comment:
ConditionalOnProperty doesn't work. EnabledIfEnvironmentVariable does work, but I'd prefer to use the application property (repository.dispute) instead of the environment variable (DISPUTE_REPOSITORY_SRC).  Or better, use Spring Profiles. Do you know how to accomplish this?

Reply:
Yeah that's a good point. I think we should be able to use Spring Profiles or Test profile to accomplish this. Just checked the spring documentation and there is an annotation called `@IfProfileValue` which might help pulling the application property value in to achieve this along with a custom properties class in Test which can be injected into the test class utilizing `@ProfileValueSourceConfiguration`. Here is the documentation: https://docs.spring.io/spring-framework/docs/current/reference/html/testing.html#integration-testing-annotations-junit4-ifprofilevalue

Reply:
Sounds good. We can look into adding test profiles in a subsequent PR.